sw/device/lib/runtime/otbn.h

```diff
@@ -118,6 +100,56 @@ typedef struct otbn {
   bool app_is_loaded;
 } otbn_t;
 
+/**
+ * Generate the prefix to add to an OTBN symbol name used on the Ibex side
+ *
+ * The result is a pointer to Ibex's rodata that should be used to initialise
+ * memory for that symbol.
+ *
+ * This is needed by the OTBN driver to support DMEM/IMEM ranges but
+ * application code shouldn't need to use this. Use the `otbn_addr_t` type and
+ * supporting macros instead.
+ */
+#define OTBN_SYMBOL_PTR(app_name, sym) _otbn_local_app_##app_name##_##sym
+
+/**
+ * Generate the prefix to add to an OTBN symbol name used on the OTBN side
+ *
+ * The result is a pointer whose integer value is the address by which the
+ * symbol should be accessed in OTBN memory.
+ *
+ * This is an internal macro used in `OTBN_DECLARE_SYMBOL_ADDR` and
+ * `OTBN_ADDR_T_INIT` but application code shouldn't need to use it directly.
+ */
+#define OTBN_SYMBOL_ADDR(app_name, sym) _otbn_remote_app_##app_name##_##sym
```

Comment:
Just to make sure I understand... does ADDR give a value in the OTBN address space or the Ibex address space? The names are kind of generic so it's not super clear what's what. If I'm understanding correctly, more appropriate names would be

```
OTBN_SYMBOL_INIT_DATA  // Pointer to initialization data in .rodata or whatever.
OTBN_SYMBOL            // Pointer to the actual mapped-in OTBN region specified by the symbol. 
```

The "ADDR" and "PTR" part seem redundant and confusing.

Reply:
ADDR = OTBN address space. PTR = Pointer to Ibex rodata.

Sorry this isn't particularly clear: maybe you could file a PR to fix the comments?